### ISSUE TITLE
Align configurations with RiboViz release 2.1 configurations

### ISSUE THREAD

**Issue by mikej888** [riboviz/example-datasets#76]
Run `riboviz.tools.upgrade_config_file` on configurations so latest RiboViz configuration parameters are present.

**Comment by ewallace** [comment]
When @3mma-mack and @swinterbourne start on 21st June, let's discuss this.

**Comment by FlicAnderson** [comment]
Planning to run the following datasets for riboviz/#355 smoke testing:

- W-Ec_2015: E.coli https://github.com/riboviz/example-datasets/blob/main/bacteria/escherichia/Woolstenhulme_2015_3_prime_mapping_RPF_2-samples_CDS_w_25utrs.yaml
- G-Hv_2020: H.volcanii: https://github.com/riboviz/example-datasets/blob/main/archaea/haloferax/Gelsinger_etal_2020_FFC_MNase_samples_CDS_w_25utrs.yaml
- L-Sc_2014: S.cerevisiae: https://github.com/riboviz/example-datasets/blob/main/fungi/saccharomyces/Lareau_2014_Replicates_RPF_3-samples_CDS_w_250utrs_config.yaml

Making new branch to edit each config to be 2.1-ready: `update-configs-76`

**Comment by 3mma-mack** [comment]
I have created an updated version of the Duncan et al Schizosaccharomyces [dataset](https://github.com/riboviz/example-datasets/tree/update-configs-76/fungi/schizosaccharomyces) and added it to the branch. I am doing a test run with an updated version of the Weinberg Saccharomyces dataset before I add that as well.

**Comment by 3mma-mack** [comment]
I have created updated versions of the Weinberg and Brar Saccharomyces datasets. Both updated files successfully ran on riboviz, so I have added them to the branch. I have left the old config files on the branch, and have named the new versions so they have `config_2-1.yaml ` to distinguish them from the old versions.

**Comment by swinterbourne** [comment]
I successfully ran the Lareau_2014_Replicates_RPF_3-samples_CDS_w_250utrs_config_2-1.yaml config. I will test run an updated version of the Jan and Gupta Saccharomyces datasets before adding those to the branch as well.

**Comment by swinterbourne** [comment]
The updated version of the Jan et al Saccharomyces dataset was successfully run and has been added to the branch with `config_2-1.yaml` at the end of the name.

**Comment by 3mma-mack** [comment]
I have created, run and tested an updated version of the Muzzey Candida dataset. This has now been pushed to example datasets.

**Comment by swinterbourne** [comment]
The Gupta dataset failed to run due to a pathway error to the barcodes sample sheet in the config file. This error has been corrected and is now rerunning. I have created an updated version of the Yu Neurospora dataset which is also currently running.

**Comment by 3mma-mack** [comment]
I am currently running the two Wallace cryptococcus updated configs, and have successfully run the Morgan Ecoli dataset, though the output files looked unusual, with no clear 3nt periodicity and a range of fragment sizes. I think bacteria usually have polycistronic mRNA, which may explain why the 3nt_periodicity plot looks the way it does, but I plan to run the original config and compare the outputs just in case. 

[3nt_periodicity.pdf](https://github.com/riboviz/example-datasets/files/6747604/3nt_periodicity.pdf)
[startcodon_ribogridbar.pdf](https://github.com/riboviz/example-datasets/files/6747607/startcodon_ribogridbar.pdf)
[read_lengths.pdf](https://github.com/riboviz/example-datasets/files/6747610/read_lengths.pdf)

**Comment by FlicAnderson** [comment]
Hi @3mma-mack - in case it might be helpful for comparison purposes, there are outputs from a run of both Cryptococcus datasets from last year on the wallace lab group drive at `/exports/csce/eddie/biology/groups/wallace_rna/20200925_W-Cn-H99_2020/` and `ls /exports/csce/eddie/biology/groups/wallace_rna/20200928_W-Cn-JEC21_2020/`. This might save you running them again?

**Comment by swinterbourne** [comment]
The Yu Neurospora dataset completed successfully and had the expected output files. The config has been pushed to example-datasets.

**Comment by 3mma-mack** [comment]
The two Wallace_2020 cryptococcus config files have been updated and run to produce identical results to those stored in `/exports/csce/eddie/biology/groups/wallace_rna/20200925_W-Cn-H99_2020`/ and `/exports/csce/eddie/biology/groups/wallace_rna/20200928_W-Cn-JEC21_2020/`. These new config files have been pushed to example-datasets

**Comment by ewallace** [comment]
It sounds like the updating is basically done as soon as these pull requests are merged. Then putting a v2.1 tag can happen after we do the riboviz/riboviz 2.1 tag.

**Comment by 3mma-mack** [comment]
Having checked and compared the updated Morgan et al config file to results produced by the original config file, which are identical, I have pushed this file to the branch. 
I think the only config that have not been upgraded and tested yet is the Ecoli Favate et al dataset, as I think the paper is still in preprint and I haven't been able to find the data. Though at this point I think it is reasonably safe to assume that upgrading the config file will not change the outputs so I may upgrade it without testing at this point in time

**Comment by ewallace** [comment]
Yes. Best to upgrade it on a branch and then ask @john-s-f to test.
